rakelib/bootstrap.rake

```diff
@@ -1,2 +1 @@
-
-task "bootstrap" => [ "vendor:all", "compile:all" ]
+task "bootstrap" => [ "vendor:gems", "vendor:all", "compile:all" ]
```

Comment:
which you did too. so you should not need to specify vendor:gems in the bootstrap task...